packages/terra-disclosure-manager/src/DisclosureManager.jsx

```diff
@@ -298,10 +298,12 @@ class DisclosureManager extends React.Component {
           name: data.content.name,
           props: data.content.props,
           component: data.content.component,
+          ...(data.content.title) && { headerAdapterData: { title: data.content.title } },
```

Comment:
> define an empty string

The spread operators will evaluate to false for an empty string. The check will need to account for `''`. 

Possible solution:
```jsx
...(data.content.title !== undefined) 
```

Reply:
I didn't realize that. Thanks!! 👍

Reply:
How granular do we want this check to be? Do we want to also allow null titles?